100-level-AsyncAPI-Intro/step2.md

```diff
@@ -0,0 +1,244 @@
+
+## Understanding AsyncAPI Components through Chan and Eve's Programming Session
+
+- **Info (Metadata about the API):**
+
+Just as the name tag on Chan's spaceship provides essential details like the spaceship's name and model, the `info` section in AsyncAPI serves a similar purpose. It provides crucial information about the API, such as the title, version, and description. This metadata helps developers understand the basic details of the API they're working with.
+
+```yml
+asyncapi: '3.0.0'
+info:
+  title: Space Communication API
+  version: '1.0.0'
+  description: API for communication between Chan's spaceship and Capuccinova
+```
```

Comment:
```suggestion
- **Info (Metadata about the API):**

Just as the name tag on Chan's spaceship provides essential details like the spaceship's name and model, the `info` section in AsyncAPI serves a similar purpose. It provides crucial information about the API, such as the title, version, and description. The `info` metadata helps developers understand the basic API details they're working with.

```yml
asyncapi: '3.0.0'
info:
  title: Space Communication API
  version: '1.0.0'
  description: API for communication between Chan's spaceship and Capuccinova
```
```